source/globalCommands.py

```diff
@@ -745,7 +745,7 @@ def script_reviewMode_previous(self,gesture):
 			speech.speakTextInfo(pos)
 		else:
 			# Translators: reported when there are no other available review modes for this object 
-			ui.reviewMessage(_("No previous review mode"))
+			ui.message(_("No previous review mode"))

```

Comment:
Why is this changed? And why is this change included in this PR?

Reply:
I simply have no idea, good catch. Reverted.